app/assets/src/components/visualizations/dendrogram/Dendogram.js

```diff
@@ -158,25 +158,34 @@ export default class Dendogram {
     }
 
     let attrName = this.options.colorGroupAttribute;
+    let absentName = this.options.colorGroupAbsentName;
 
     // Set up all attribute values. Colors end up looking like:
-    // Uncolored (grey) | Real values.. | Absent attribute color (e.g. for External Sources)
+    // Uncolored (grey) | Absent attribute color (e.g. for NCBI References) + Real values..
```

Comment:
what do you mean by real values?

Reply:
values that were actually seen? I could change to "Values found"